climate_emotions_map/app.py

```diff
@@ -120,10 +120,12 @@ def reset_impact_select(question_value):
 @callback(
     Output("map-title", "children"),
     Input("impact-select", "value"),
+    State("impact-select", "data"),
 )
-def update_map_title(impact):
+def update_map_title(impact, opts):
```

Comment:
:cherries: should we call `opts` `options` here? Pretty close to `opt` below, and took me a second to see that there was a list comprehension going on.

Reply:
also seems to be named `options` elsewhere, no?